cfn-resources/autogen/README.md

```diff
@@ -1,67 +1,68 @@
-# Auto Generate cfn template and generic GO Code
+# Using autogen 
```

Comment:
During the onboarding to CFN we [documented](https://wiki.corp.mongodb.com/display/MMS/CFN+Development+Guide#CFNDevelopmentGuide-AutomatedApproach) these limitations for the autogen script:

 ```
Limitations of the current implementations: [INTMDB-562](https://jira.mongodb.org/browse/INTMDB-562)

The schema-gen code has the following problem which would require changing the logic of the script:
Example: we have the property connectionStrings.privateEndpoint in [createCluster ](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Multi-Cloud-Clusters/operation/createCluster)(Response 201), which ref to ApiAtlasClusterDescriptionConnectionStringsPrivateEndpointView . Unfortunately, the definition of ApiAtlasClusterDescriptionConnectionStringsPrivateEndpointView is not added to the JSON schema file.
the code does not traverse a view property tree but it stops at the 2nd level

the autogen code adds additional boilercode to the code generated by cfn generate but it still requires several manual changes
```

I am wondering if this PR addresses these limitations as well

Reply:
no this PR doesn't address that limitation